answer/56miyano/html/2_anchor.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <a href="https://isc-makeit.github.io/Make-IT-homepage/">Make ITホームページ</a>
+<br>
+<a href="https://isc-makeit.github.io/Make-IT-homepage/" target="blank" >Make ITホームページ</a>
```

Comment:
```suggestion
     <br>
     <a href="https://isc-makeit.github.io/Make-IT-homepage/" target="blank" >Make ITホームページ</a>
````
インデントは前後のタグに合わせましょう！

Reply:
aタグのtarget blankをつけるときは noopenerもつけると良いです:eyes:

https://qiita.com/memolog/items/9913b0067e4e6883ed17